mods/gen2/items.js

```diff
@@ -30,8 +30,8 @@ exports.BattleItems = {
 				return 3;
 			}
 		},
-		desc: "If holder is a Chansey, it overrides all crit boosting effects, and sets Chansey's crit ratio to 3 (crit chance = 1/3).",
-	},
+		desc: "If holder is a Chansey, it overrides all crit boosting effects, and sets Chansey's crit ratio to 3 (crit chance = 1/3).",	
+		shortDesc: "If holder is a Chansey, it removes all crit boosting effects, and crit ratio = 1/3",
```

Comment:
"it removes all crit boosting effects" is also wrong.

The phrasing I'd use would be

`If held by a Chansey, its crit rate is set to 3, overriding all other crit rate changes.`

For a long description, I might clarify that to

`If held by a Chansey, its crit rate is set to 3 (for a crit chance of 33%), overriding all other crit rate changes.`